mentors/elixir/e.json

```diff
@@ -1 +1,8 @@
-[]
+[{
+  "github_username": "emadb",
+  "name": "Emanuele DelBono",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I'm a developer passionate about clean code that falled in love with Elixir"
```

Comment:
indent lines 2-7 by two spaces so they are two spaces indented in from `{` and `}`.